gcp-observability/src/main/java/io/grpc/gcp/observability/ObservabilityConfigImpl.java

```diff
@@ -139,21 +139,21 @@ private void parseConfig(Map<String, ?> config) {
   private EventType convertEventType(String val) {
```

Comment:
This can be `static`?

Reply:
We don't even need this method. You can just use `EventType.valueOf(val)` instead of calling this method

Reply:
Done